apps/mobile/src/features/account-list/account-list.tsx

```diff
@@ -23,13 +24,17 @@ export function AccountList({ accounts, onPress, showWalletInfo }: AccountListPr
       {wallet => (
         <AccountListItem
           accountName={account.name}
-          address={t`Address`}
-          balance={t`$1234`}
+          address={
```

Comment:
Thanks for updating this 👏  

Not for this PR and aside from your work, I was looking at this file and I wonder could we improve how we do things. 

We map over `accounts` and then load `wallet` for each account so we can show the `wallet.name`. Maybe we could structure this better? Or add a selector to get the `wallet.name` based on the account

```
return accounts.map(account => (
    <WalletLoader fingerprint={account.fingerprint} key={account.id}>
      {wallet => (
        <AccountListItem
          accountName={account.name}
          ...
          walletName={showWalletInfo ? wallet.name : undefined}
        />
      )}
    </WalletLoader>
  ));
```

Reply:
Agree here, been thinking on this too bc I'm just now starting to interact with code dealing with wallets and accounts.